frontend/.pa11yci-desktop.json

```diff
@@ -8,6 +8,9 @@
       "ignoreHTTPSErrors": true,
       "args": ["--disable-dev-shm-usage", "--no-sandbox"]
     },
+    "headers": {
+      "Cookie": "_ff={%22showSearchV0%22:true}"
+    },
```

Comment:
This is actually the only way that it works. The idea we had of going to the FF page and clicking didn't work because it would execute on every loop and cause the test to stay on the FF page. 

The last page we go to is the search page, which has all the results. You can verify it on this branch. it will be the last saved screenshot with all the tests. I attached a screenshot to this PR.